app/src/main/java/com/cornellappdev/transit/ui/components/RouteCell.kt

```diff
@@ -0,0 +1,682 @@
+package com.cornellappdev.transit.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Face
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.material.icons.outlined.KeyboardArrowLeft
+import androidx.compose.material.icons.outlined.KeyboardArrowRight
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.transit.R
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.cornellappdev.transit.ui.theme.IconGrey
+import com.cornellappdev.transit.ui.theme.LateRed
+import com.cornellappdev.transit.ui.theme.LiveGreen
+import com.cornellappdev.transit.ui.theme.MetadataGrey
+import com.cornellappdev.transit.ui.theme.PrimaryText
+import com.cornellappdev.transit.ui.theme.SecondaryText
+import com.cornellappdev.transit.ui.theme.TransitBlue
+import com.cornellappdev.transit.ui.theme.sfProDisplayFamily
+import com.cornellappdev.transit.ui.theme.sfProTextFamily
+
+//Invariant: if transport is WalkOnly, lateness must be None <- maybe can enforce this somehow idk
+@Composable
+fun RouteCell(transport: Transport) {
```

Comment:
Not exactly sure where to comment this, but there definitely seems to be some minor disparities between this and figma. For example, you have a bit too much space between the On Time stuff and the header:

Yours:
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/f16b0799-cc26-4422-80d8-35f395257388)

Design:
![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/58c670c9-6bce-42de-9118-e21d1e96b8ee)

Also, the caret on the right is too bold.

Try to fix up these small design inconsistencies! They're hard to spot after merging.